tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true
```

Comment:
Please also update the npmignore file to avoid publishing the new sourcemap

Reply:
Done